app/helpers/reports/regions_helper.rb

```diff
@@ -36,4 +36,12 @@ def percentage_or_na(value, options)
   def cohort_report_type(period)
     "#{period.type.to_s.humanize}ly report"
   end
+
+  def follow_ups_definition
```

Comment:
The naming might get weird here, since it's called `follow_up_patients*` in the locales. Let's remember to clean this up once we switch